context.go

```diff
@@ -153,3 +153,51 @@ func (p *Context) YAP(code int, yapFile string, data interface{}) {
 		log.Panicln("YAP:", err)
 	}
 }
+
+func (p *Context) STREAM(code int, mime string, read io.ReadCloser, buf []byte) {
```

Comment:
`read` should be an `io.Reader` instead of `io.ReadCloser`, as the `Context.Stream`'s caller should be responsible for closing it. We shouldn't assume ownership of the `read`.

Reply:
ok, use io.Reader instead of io.ReadCloser